app/helpers/database_service.py

```diff
@@ -0,0 +1,156 @@
+
+from mysql.connector import connect, Error
+import os
+
+
+class DatabaseService:
+    # def __init__(self):
+    def create_connection(self):
+        try:
+            super_connection = connect(
+                host=os.getenv('ADMIN_MYSQL_HOST'),
+                user=os.getenv('ADMIN_MYSQL_USER'),
+                password=os.getenv('ADMIN_MYSQL_PASSWORD'),
+                port=os.getenv('ADMIN_MYSQL_PORT', '')
+            )
+            return super_connection
+        except Error as e:
+            print(e)
+            return False
+
+    def create_db_connection(self, user=None, password=None, db_name=None):
+        try:
+            self.user_connection = connect(
+                host=os.getenv('ADMIN_MYSQL_HOST'),
+                user=user,
+                password=password,
+                port=os.getenv('ADMIN_MYSQL_PORT', ''),
+                database=db_name
+            )
+            return True
+        except Error as e:
+            print(e)
+            return False
+
+    # Create or check user exists database
+    def create_db(self, db_name=None, user=None, password=None):
+        try:
+            connection = self.create_connection()
+            if not connection:
+                return False
+            cursor = connection.cursor()
+            cursor.execute(f"CREATE DATABASE {db_name}")
+            if self.create_user(user=user, password=password):
+                cursor.execute(
+                    f"GRANT ALL PRIVILEGES ON {db_name}.* To '{user}'")
+            return True
+        except Error as e:
+            print(e)
+            return False
+        finally:
+            if not connection:
+                return False
+            if (connection.is_connected()):
+                cursor.close()
+                connection.close()
+
+    # create database user
+    def create_user(self, user=None, password=None):
```

Comment:
the create user methods fails on each attempt. I tried it when creating a database, the db is created but the user is not.

Reply:
I also test it separately and it didn't create the user either.

Reply:
It was just a difference in environments for testing.